cluster-autoscaler/metrics/metrics.go

```diff
@@ -50,6 +50,12 @@ const (
 	LogLongDurationThreshold = 5 * time.Second
 )
 
+const (
```

Comment:
Right now to be able to document the whole block of consts at once. (unless there's another way to do it)